packages/react-search-ui-views/src/__tests__/view-helpers/formatResult.test.ts

```diff
@@ -1,4 +1,53 @@
 import { formatResult } from "../../view-helpers";
+import { cleanValueWrappers } from "../../view-helpers/formatResult";
+
+it("cleanValueWrappers", () => {
```

Comment:
Is it worthwhile to add a test that verifies that everything that needs to be escaped is properly run through `getEscapedField`?